test/extended/apibinding/apibinding.go

```diff
@@ -0,0 +1,44 @@
+package apibinding
+
+import (
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+
+	exutil "github.com/kcp-dev/kcp-tests/test/extended/util"
+)
+
+var _ = g.Describe("[sig-apibinding]", func() {
+	defer g.GinkgoRecover()
+
+	var (
+		k = exutil.NewCLIWithWorkSpace("kcp-apibinding")
+	)
+
+	g.It("Author:pewang-Critical-[Smoke] Create an apibinding attach a workspace to the exist shared compute should work well", func() {
+		// Share compute could only access from specific test environments
+		// Skip for none supported test environments
+		exutil.PreCheckEnvSupport(k, "kcp-stable.apps.kcp-internal", "kcp-unstable.apps.kcp-internal")
+
+		g.By("# Create an apibinding attach a workspace to the exist shared compute")
+		myAPIBing := NewAPIBinding(SetAPIBindingReferencePath("root:redhat-acm-compute"), SetAPIBindingReferenceExportName("kubernetes"))
+		myAPIBing.Create(k)
```

Comment:
s/Bing/Binding/

Reply:
s/# Create an apibinding attach a workspace to the exist shared compute/# Create an apibinding for the shared compute provided by acm"

Reply:
Fixed.